demo-app/src/open-truss/configs/4-trino-demo.yaml

```diff
@@ -1,9 +1,27 @@
 workflow:
   version: 1
+  signals:
+    results: Records
   frames:
     - frame:
-      data:
-        query: select * from  tpch.sf1.customer limit 10
-        source: trino-demo
       view:
-        component: TrinoDemo
+        component: OTUqiDataProvider
+        props:
+          source: trino-demo
+          query: >
+            SELECT name, phone, acctbal, comment
```

Comment:
@jonmagic I was able to get this to work. The previous columns we were using (e.g. `Column`, `Type`, `Extra`) just didn't exist on this table at all and it wasn't a capitalization issue. Did you accidentally get those column names from somewhere else?